vnext/Microsoft.ReactNative/Modules/NativeUIManager.cpp

```diff
@@ -153,9 +153,10 @@ NativeUIManager::NativeUIManager(winrt::Microsoft::ReactNative::ReactContext con
   m_yogaConfig = YGConfigNew();
   if (React::implementation::QuirkSettings::GetUseWebFlexBasisBehavior(m_context.Properties()))
     YGConfigSetExperimentalFeatureEnabled(m_yogaConfig, YGExperimentalFeatureWebFlexBasis, true);
+  auto errata = YGErrataAll;
   if (React::implementation::QuirkSettings::GetMatchAndroidAndIOSStretchBehavior(m_context.Properties()))
-    YGConfigSetErrata(m_yogaConfig, YGErrataStretchFlexBasis);
-
+    errata &= ~YGErrataStretchFlexBasis;
```

Comment:
I think this is reversed.

Yoga was changed to be conformant by default, and a specific YGErrata option is a compatibility option. 

So, YGErrataAll means “prefer full compatibility”, and sets YGErrataStretchFlexBasis as the equivalent of UseLegacyStretchBehaviour.

So, this line effectively disables UseLegacyStretchBehaviour, instead of enabling it.

There is also a convenience YGErrataClassic to match the defaults of Yoga 1.x (UseLegacyStretchBehaviour is kept disabled, but all future errata are enabled).

Reply:
We’re not helped by documentation for the enums being lost in a codegen step, but I’m beginning to wonder if I should consider changing these names again before we put out the first Yoga 2 release. They are not seeming intuitive.